src/runtime/core/logger/logger_manager.cc

```diff
@@ -198,16 +238,34 @@ const LoggerProxy& LoggerManager::GetLoggerProxy(std::string_view logger_name) {
   return *(emplace_ret.first->second);
 }
 
+/**
+ * @brief Retrieve the log levels of all loggers in a thread-safe manner.
+ * @return A map of logger names and their corresponding log levels.
+ *
+ * This function returns the current log levels of all registered loggers.
+ * It is thread-safe and can be called concurrently from multiple threads.
+ */
 std::unordered_map<std::string, aimrt_log_level_t> LoggerManager::GetAllLoggerLevels() const {
   std::unordered_map<std::string, aimrt_log_level_t> result;
+  std::lock_guard<std::mutex> lock(logger_proxy_map_mutex_);  // 添加锁保护
```

Comment:
No lock is required here. This method can be restricted to the start phase only.